packages/common/src/services/integrations/jwp/JWPAPIService.ts

```diff
@@ -2,6 +2,7 @@ import { inject, injectable } from 'inversify';
 
```

Comment:
We can revert this as it was, and define the needed methods in the ApiService. They need to be usable with other integrations as well. The access bridge host can be used from the env, as it will have only one environment.